modules/osd-create-cluster-ccs.adoc

```diff
@@ -199,7 +199,7 @@ endif::osd-on-aws[]
 ifdef::osd-on-gcp[]
 GCP
 endif::osd-on-gcp[]
-VPC, provide your *Virtual Private Cloud (VPC) subnet settings* and select *Next*.
+VPC, provide your *Virtual Private Cloud (VPC) subnet settings* and select *Next*. This process requires the creation of the Cloud network address translation (NAT) as well as a Cloud router. See the additional resources for information on Cloud NATs and Google VPCs.
```

Comment:
> This process requires

Who is responsible for creating these things?

Reply:
@mburke5678 it is assumed that whomever is creating this cluster with a VPC would have those resources created, either themselves or someone in their organization.

Reply:
How about: You must have created the Cloud network address translation (NAT) as well as a Cloud router

Reply:
@mburke5678 I dig it! Updated the PR